src/lib/rmt/wizard_rmt_service_page.rb

```diff
@@ -69,15 +74,45 @@ def back_handler
 
   def run
     render_content
-    ok = false
+    service_handler('rmt-server')
+    service_handler('nginx')
+    event_loop
+  end
+
+  def service_handler(service_name)
+    handler = handlers.fetch(service_name)
+    handler.service_handler
+  end
+end
+
+class NginxServiceHandler
```

Comment:
Copypasta of `RMTServerServiceHandler`? :grimacing:

Reply:
Actually I tried refactoring as good as possible.